<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LiveJournalCfgBase</class>
 <widget class="QWidget" name="LiveJournalCfgBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>11</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabLiveJournal">
     <widget class="QWidget" name="tabLogin">
      <attribute name="title">
       <string>&amp;LiveJournal</string>
      </attribute>
      <layout class="QGridLayout">
       <property name="margin">
        <number>11</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="TextLabel1">
         <property name="text">
          <string>Username:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="edtName"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblPassword">
         <property name="text">
          <string>Password:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="edtPassword">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="LinkLabel" name="lblLnk" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>&amp;Network</string>
      </attribute>
      <layout class="QGridLayout">
       <property name="margin">
        <number>11</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="edtServer"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="TextLabel3">
         <property name="text">
          <string>Server:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="TextLabel4">
         <property name="text">
          <string>Path:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="edtPath"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="TextLabel5">
         <property name="text">
          <string>Port:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="edtPort">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>&amp;Options</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>11</number>
       </property>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="TextLabel1_2">
           <property name="text">
            <string>Check interval:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="edtInterval"/>
         </item>
         <item>
          <widget class="QLabel" name="TextLabel2_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>minutes</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="chkFastServer">
         <property name="text">
          <string>Use &amp;fast server - only for paying customers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkUseFormatting">
         <property name="text">
          <string>Use formatted messages</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkUseSignature">
         <property name="text">
          <string>Add signature to all messages</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="edtSignature"/>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LinkLabel</class>
   <extends>QWidget</extends>
   <header>simgui/linklabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
